xonsh/built_ins.py

```diff
@@ -185,12 +185,23 @@ def expand_path(s):
     return os.path.expanduser(s)
 
 
+WINDOWS_DRIVE_MATCHER = re.compile(r'^\w:'
+
+                                   )
```

Comment:
Sorry -- I meant that the closing paren (on line 190) should be moved up so the regex is on one line.